### DISCUSSION TITLE
[Envoy] How the rate limiter service defines the uniqueness of the addresses

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#334] [Q&A] [answered]
### Which section?

Marketplace

### Which is the version?

v11.7.1

### Description

Hi guys,
in the RLS plugin public Mia-Platform documentation, available at [this link](https://docs.mia-platform.eu/docs/runtime_suite/envoy-api-gateway/filters#rate-limit), I read the following sentence:
> applies a rate limit of 1000 requests per second on each unique remote address.

How does the RLS plugin define the uniqueness of the addresses? Is IP-based?
Thanks.

**Comment by fredmaggiowski** [comment]
Hey, you can find further information about the plugin here: https://github.com/envoyproxy/ratelimit

**Reply by frengooo** [reply]
Thanks @fredmaggiowski, I'll add this link in the documentation too.

**Reply by fredmaggiowski** [reply]
[it is already linked](https://docs.mia-platform.eu/docs/runtime_suite/envoy-api-gateway/filters#rate-limit) :)

**Comment by davidebianchi** [comment, marked as answer]
Yes, by default the Console configure the plugin to match the remote address. So, this check is ip based.

**Reply by frengooo** [reply]
Thanks, I'll open  a PR  to add this information (maybe 😄 ).
